From a15b75e9f307c0206e1f3298cc4f3fafc6f2bd16 Mon Sep 17 00:00:00 2001
From: amidevous <amidevous@users.noreply.github.com>
Date: Sun, 11 Jun 2023 12:47:40 +0200
Subject: [PATCH] Update local.mk

---
 bin/local.mk | 32 ++++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/bin/local.mk b/bin/local.mk
index 97b38db34..7bdfc5188 100644
--- a/bin/local.mk
+++ b/bin/local.mk
@@ -18,21 +18,21 @@
 ##  The automake and aclocal scripts.  ##
 ## ----------------------------------- ##
 
-bin_SCRIPTS = %D%/automake %D%/aclocal
+bin_SCRIPTS = bin/automake bin/aclocal
 nodist_noinst_SCRIPTS += \
-  %D%/aclocal-$(APIVERSION) \
-  %D%/automake-$(APIVERSION)
+  bin/aclocal-$(APIVERSION) \
+  bin/automake-$(APIVERSION)
 
 CLEANFILES += \
   $(bin_SCRIPTS) \
-  %D%/aclocal-$(APIVERSION) \
-  %D%/automake-$(APIVERSION)
+  bin/aclocal-$(APIVERSION) \
+  bin/automake-$(APIVERSION)
 
 # Used by maintainer checks and such.
-automake_in = $(srcdir)/%D%/automake.in
-aclocal_in  = $(srcdir)/%D%/aclocal.in
-automake_script = %D%/automake
-aclocal_script  = %D%/aclocal
+automake_in = $(srcdir)/bin/automake.in
+aclocal_in  = $(srcdir)/bin/aclocal.in
+automake_script = bin/automake
+aclocal_script  = bin/aclocal
 
 AUTOMAKESOURCES = $(automake_in) $(aclocal_in)
 TAGS_FILES += $(AUTOMAKESOURCES)
@@ -60,9 +60,9 @@ uninstall-hook:
 
 # These files depend on Makefile so they are rebuilt if $(VERSION),
 # $(datadir) or other do_subst'ituted variables change.
-%D%/automake: %D%/automake.in
-%D%/aclocal: %D%/aclocal.in
-%D%/automake %D%/aclocal: Makefile
+bin/automake: bin/automake.in
+bin/aclocal: bin/aclocal.in
+bin/automake bin/aclocal: Makefile
 	$(AM_V_GEN)rm -f $@ $@-t $@-t2 \
 	  && $(MKDIR_P) $(@D) \
 ## Common substitutions.
@@ -72,12 +72,12 @@ uninstall-hook:
 ## comments, and that is perfectly legit.
 	  && chmod a+x,a-w $@-t && mv -f $@-t $@
 
-%D%/aclocal-$(APIVERSION): %D%/aclocal
+bin/aclocal-$(APIVERSION): bin/aclocal
 	$(AM_V_GEN) rm -f $@; \
-	$(LN) %D%/aclocal $@
+	$(LN) bin/aclocal $@
 
-%D%/automake-$(APIVERSION): %D%/automake
+bin/automake-$(APIVERSION): bin/automake
 	$(AM_V_GEN) rm -f $@; \
-	$(LN) %D%/automake $@
+	$(LN) bin/automake $@
 
 # vim: ft=automake noet

From 2ee4500bc04e5e538ba501e61660e9368217e96a Mon Sep 17 00:00:00 2001
From: amidevous <amidevous@users.noreply.github.com>
Date: Sun, 11 Jun 2023 12:51:42 +0200
Subject: [PATCH] Update local.mk

---
 contrib/t/local.mk | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/contrib/t/local.mk b/contrib/t/local.mk
index 6bebb7063..eb6eae305 100644
--- a/contrib/t/local.mk
+++ b/contrib/t/local.mk
@@ -19,7 +19,7 @@
 ## -------------------------------- ##
 
 contrib_TESTS = \
-  %D%/parallel-tests-html.sh \
-  %D%/parallel-tests-html-recursive.sh \
-  %D%/help-multilib.sh \
-  %D%/multilib.sh
+  contrib/t/parallel-tests-html.sh \
+  contrib/t/parallel-tests-html-recursive.sh \
+  contrib/t/help-multilib.sh \
+  contrib/t/multilib.sh


From d52388e017a4953a1985e72900d0a3178a6b6c61 Mon Sep 17 00:00:00 2001
From: amidevous <amidevous@users.noreply.github.com>
Date: Sun, 11 Jun 2023 12:26:20 +0200
Subject: [PATCH] Update local.mk

---
 doc/local.mk | 44 ++++++++++++++++++++++----------------------
 1 file changed, 22 insertions(+), 22 deletions(-)

diff --git a/doc/local.mk b/doc/local.mk
index e76c04f5d..4975d21f4 100644
--- a/doc/local.mk
+++ b/doc/local.mk
@@ -18,36 +18,36 @@
 ##  Documentation.  ##
 ## ---------------- ##
 
-info_TEXINFOS = %D%/automake.texi %D%/automake-history.texi
-doc_automake_TEXINFOS = %D%/fdl.texi
-doc_automake_history_TEXINFOS = %D%/fdl.texi
+info_TEXINFOS = doc/automake.texi doc/automake-history.texi
+doc_automake_TEXINFOS = doc/fdl.texi
+doc_automake_history_TEXINFOS = doc/fdl.texi
 
 man1_MANS = \
-  %D%/aclocal.1 \
-  %D%/automake.1 \
-  %D%/aclocal-$(APIVERSION).1 \
-  %D%/automake-$(APIVERSION).1
+  doc/aclocal.1 \
+  doc/automake.1 \
+  doc/aclocal-$(APIVERSION).1 \
+  doc/automake-$(APIVERSION).1
 
 $(man1_MANS): $(top_srcdir)/configure.ac
 
 CLEANFILES += $(man1_MANS)
 # XXX: This script should be updated with 'fetch' target.
-EXTRA_DIST += %D%/help2man
+EXTRA_DIST += doc/help2man
 
 update_mans = \
   $(AM_V_GEN): \
-    && $(MKDIR_P) %D% \
-    && ./pre-inst-env $(PERL) $(srcdir)/%D%/help2man --output=$@
+    && $(MKDIR_P) doc \
+    && ./pre-inst-env $(PERL) $(srcdir)/doc/help2man --output=$@
 
-%D%/aclocal.1 %D%/automake.1:
+doc/aclocal.1 doc/automake.1:
 	$(AM_V_GEN): \
-	  && $(MKDIR_P) %D% \
+	  && $(MKDIR_P) doc \
 	  && f=`echo $@ | sed 's|.*/||; s|\.1$$||; $(transform)'` \
 	  && echo ".so man1/$$f-$(APIVERSION).1" > $@
 
-%D%/aclocal-$(APIVERSION).1: $(aclocal_script) lib/Automake/Config.pm
+doc/aclocal-$(APIVERSION).1: $(aclocal_script) lib/Automake/Config.pm
 	$(update_mans) aclocal-$(APIVERSION)
-%D%/automake-$(APIVERSION).1: $(automake_script) lib/Automake/Config.pm
+doc/automake-$(APIVERSION).1: $(automake_script) lib/Automake/Config.pm
 	$(update_mans) automake-$(APIVERSION)
 
 ## ---------------------------- ##
@@ -55,11 +55,11 @@ update_mans = \
 ## ---------------------------- ##
 
 amhello_sources = \
-  %D%/amhello/configure.ac \
-  %D%/amhello/Makefile.am \
-  %D%/amhello/README \
-  %D%/amhello/src/main.c \
-  %D%/amhello/src/Makefile.am
+  doc/amhello/configure.ac \
+  doc/amhello/Makefile.am \
+  doc/amhello/README \
+  doc/amhello/src/main.c \
+  doc/amhello/src/Makefile.am
 
 amhello_configury = \
   aclocal.m4 \
@@ -73,7 +73,7 @@ amhello_configury = \
   src/Makefile.in
 
 dist_noinst_DATA += $(amhello_sources)
-dist_doc_DATA = $(srcdir)/%D%/amhello-1.0.tar.gz
+dist_doc_DATA = $(srcdir)/doc/amhello-1.0.tar.gz
 
 setup_autotools_paths = { \
   ACLOCAL=aclocal-$(APIVERSION) && export ACLOCAL \
@@ -88,9 +88,9 @@ setup_autotools_paths = { \
 
 # We depend on configure.ac so that we regenerate the tarball
 # whenever the Automake version changes.
-$(srcdir)/%D%/amhello-1.0.tar.gz: $(amhello_sources) $(srcdir)/configure.ac
+$(srcdir)/doc/amhello-1.0.tar.gz: $(amhello_sources) $(srcdir)/configure.ac
 	$(AM_V_GEN)tmp=amhello-output.tmp \
-	  && $(am__cd) $(srcdir)/%D%/amhello \
+	  && $(am__cd) $(srcdir)/doc/amhello \
 	  && : Make our	aclocal and automake avaiable before system ones. \
 	  && $(setup_autotools_paths) \
 	  && ( \

From d17232c843f8011f46990266fd734d9eb5266424 Mon Sep 17 00:00:00 2001
From: amidevous <amidevous@users.noreply.github.com>
Date: Sun, 11 Jun 2023 12:59:10 +0200
Subject: [PATCH] Update local.mk

---
 lib/Automake/local.mk | 50 +++++++++++++++++++++----------------------
 1 file changed, 25 insertions(+), 25 deletions(-)

diff --git a/lib/Automake/local.mk b/lib/Automake/local.mk
index fbb21f3f1..de8e845d7 100644
--- a/lib/Automake/local.mk
+++ b/lib/Automake/local.mk
@@ -21,36 +21,36 @@
 perllibdir = $(pkgvdatadir)/Automake
 
 dist_perllib_DATA = \
-  %D%/ChannelDefs.pm \
-  %D%/Channels.pm \
-  %D%/Condition.pm \
-  %D%/Configure_ac.pm \
-  %D%/DisjConditions.pm \
-  %D%/FileUtils.pm \
-  %D%/General.pm \
-  %D%/Getopt.pm \
-  %D%/Item.pm \
-  %D%/ItemDef.pm \
-  %D%/Language.pm \
-  %D%/Location.pm \
-  %D%/Options.pm \
-  %D%/Rule.pm \
-  %D%/RuleDef.pm \
-  %D%/Variable.pm \
-  %D%/VarDef.pm \
-  %D%/Version.pm \
-  %D%/XFile.pm \
-  %D%/Wrap.pm
+  lib/Automake/ChannelDefs.pm \
+  lib/Automake/Channels.pm \
+  lib/Automake/Condition.pm \
+  lib/Automake/Configure_ac.pm \
+  lib/Automake/DisjConditions.pm \
+  lib/Automake/FileUtils.pm \
+  lib/Automake/General.pm \
+  lib/Automake/Getopt.pm \
+  lib/Automake/Item.pm \
+  lib/Automake/ItemDef.pm \
+  lib/Automake/Language.pm \
+  lib/Automake/Location.pm \
+  lib/Automake/Options.pm \
+  lib/Automake/Rule.pm \
+  lib/Automake/RuleDef.pm \
+  lib/Automake/Variable.pm \
+  lib/Automake/VarDef.pm \
+  lib/Automake/Version.pm \
+  lib/Automake/XFile.pm \
+  lib/Automake/Wrap.pm
 
-nodist_perllib_DATA = %D%/Config.pm
+nodist_perllib_DATA = lib/Automake/Config.pm
 CLEANFILES += $(nodist_perllib_DATA)
 
-%D%/Config.pm: %D%/Config.in Makefile
+lib/Automake/Config.pm: lib/Automake/Config.in Makefile
 	$(AM_V_at)rm -f $@ $@-t
-	$(AM_V_at)$(MKDIR_P) %D%
+	$(AM_V_at)$(MKDIR_P) lib/Automake
 	$(AM_V_GEN)in=Config.in \
-	  && $(do_subst) <$(srcdir)/%D%/Config.in >$@-t
+	  && $(do_subst) <$(srcdir)/lib/Automake/Config.in >$@-t
 	$(generated_file_finalize)
-EXTRA_DIST += %D%/Config.in
+EXTRA_DIST += lib/Automake/Config.in
 
 # vim: ft=automake noet

